src/streams.c

```diff
@@ -98,6 +98,59 @@ Int READ_COMMAND ( void )
     return 1;
 }
 
+Obj FuncREAD_ALL_COMMANDS( Obj self, Obj stream, Obj echo )
+{
+    ExecStatus Status;
+    Int ResultCount, ResultCapacity;
+    Obj Result, ResultList;
```

Comment:
ahh, please don't hit me, but: we also tend to start variable names in lower case, uppercase is for functions... :)